src/1_create_canonical.jl

```diff
@@ -112,6 +112,7 @@ output_features .= ifelse.(ismissing.(output_features), "NA", output_features)
 # (GeoDataFrames cannot handle these automatically yet)
 string_cols = contains.(string.(typeof.(eachcol(output_features))), "String")
 output_features[!, contains.(string.(typeof.(eachcol(output_features))), "String")] .= String.(output_features[:, string_cols])
+output_features.GBRMPA_ID = ifelse.((output_features.GBRMPA_ID .== "20198"), "20-198", output_features.GBRMPA_ID) #edit incorrect GBRMPA_ID
```

Comment:
Make the comment more explicit with spacing to make the change stand out, and follow style guide for comments (space after the hash is required)

```suggestion

# Correct typo in GBRMPA_ID list
output_features.GBRMPA_ID = ifelse.((output_features.GBRMPA_ID .== "20198"), "20-198", output_features.GBRMPA_ID) 
```